### PR TITLE
fix: minor jumpstart dev ex improvements

### PR DESCRIPTION
**Pull request by evakravi**
*Issue #, if available:*

*Description of changes:*
- add `sagemaker.model.Model.create()` support for `accept_eula` argument
- Remove "Your model is not compiled" warning for JumpStart models that use neuron instances
- Ensure logging wildcard warning happens a single time for gated models
- improve error message for vulnerable/deprecated models to suggest SDK upgrade
- removing all code associated `generated_text_response_key`, since it is not being used in favor of another metadata field
- Remove excessive logging for `notebook_utils` (and ability to disable all JS logging)
- add support for model-specific `info` logging message

*Testing done:*
Unit tests added and passing

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/model.py [hunk 1]
```diff
@@ -635,7 +648,7 @@ def prepare_container_def(
             self.repacked_model_data or self.model_data,
             deploy_env,
             image_config=self.image_config,
-            accept_eula=getattr(self, "accept_eula", None),
+            accept_eula=accept_eula or getattr(self, "accept_eula", None),
```
Comment (chrstfu) [inline src/sagemaker/model.py:651]: question: Why not adhere the changes to `create()` to the current pattern outlined in [deploy()](https://github.com/aws/sagemaker-python-sdk/blob/ef8dd31d6b5e915c746788e9ec3cdb308f227dd2/src/sagemaker/model.py#L1366)? I'd recommend we stick to either passing in the value or through using `getattr`, but not both. wdyt?

File: src/sagemaker/multidatamodel.py [hunk 2]
```diff
@@ -121,7 +121,11 @@ def __init__(
             )
 
     def prepare_container_def(
-        self, instance_type=None, accelerator_type=None, serverless_inference_config=None
+        self,
+        instance_type=None,
+        accelerator_type=None,
+        serverless_inference_config=None,
+        accept_eula=None,
```
Comment (chrstfu) [inline src/sagemaker/multidatamodel.py:128]: Another plus to adhering to `getattr` is now you won't need to make every modification here

File: src/sagemaker/jumpstart/payload_utils.py [hunk 3]
```diff
@@ -115,80 +115,6 @@ def _construct_payload(
     return payload_to_use
 
 
-def _extract_generated_text_from_response(
```
Comment (chrstfu) [inline src/sagemaker/jumpstart/payload_utils.py:118]: nit: Could you specify why you are removing this in the PR description?

File: src/sagemaker/jumpstart/types.py [hunk 4]
```diff
@@ -371,7 +370,6 @@ def from_json(self, json_obj: Optional[Dict[str, Any]]) -> None:
         self.content_type = json_obj["content_type"]
         self.body = json_obj["body"]
         accept = json_obj.get("accept")
-        self.generated_text_response_key = json_obj.get("generated_text_response_key")
```
Comment (chrstfu) [inline src/sagemaker/jumpstart/types.py:374]: nit: same as above

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: b9a8ef19757862699609280067461b833ae1060a
* Result: FAILED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=aa1f25ed-bd96-462f-9666-2fb70505bfd0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/jumpstart/utils.py [hunk 5]
```diff
@@ -109,7 +109,8 @@ def get_jumpstart_gated_content_bucket(
     accessors.JumpStartModelsAccessor.set_jumpstart_gated_content_bucket(gated_bucket_to_return)
 
     if gated_bucket_to_return != old_gated_content_bucket:
-        accessors.JumpStartModelsAccessor.reset_cache()
+        if old_gated_content_bucket is not None:
```
Comment (chrstfu) [inline src/sagemaker/jumpstart/utils.py:112]: nit: This seems to be copied from [get_jumpstart_content_bucket()](https://github.com/aws/sagemaker-python-sdk/blob/ef8dd31d6b5e915c746788e9ec3cdb308f227dd2/src/sagemaker/jumpstart/utils.py#L119). Can we pass in these bucket names in and merge these two functions?

File: tests/unit/sagemaker/jumpstart/model/test_model.py [hunk 6]
```diff
@@ -170,6 +170,43 @@ def test_prepacked(
             ],
         )
 
+    @mock.patch("sagemaker.model.LOGGER.warning")
+    @mock.patch("sagemaker.utils.sagemaker_timestamp")
+    @mock.patch("sagemaker.session.Session.endpoint_from_production_variants")
+    @mock.patch("sagemaker.session.Session.create_model")
```
Comment (chrstfu) [inline tests/unit/sagemaker/jumpstart/model/test_model.py:176]: nit: Can we mock these functions from the `mock_session` patch?

**Review by chrstfu — COMMENTED** [review]
Nice PR! 

A quick question, I may have gotten confused by the wording of  `Ensure logging wildcard warning happens a single time for gated models`. IIUC this means we want to log only once? If that's the case, could you point me to the change that enables a single log? Thanks!

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: b9a8ef19757862699609280067461b833ae1060a
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=9074005a-14b6-431f-8ae6-bbe189e2e20e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/model.py (hunk 1 above)
Comment (chrstfu) [reply src/sagemaker/model.py:651]: imo we should pass it in and modify `deploy()` to match but would definitely want to have the group weigh in on this

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: b9a8ef19757862699609280067461b833ae1060a
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=f742796f-2f7b-485f-b052-ee23f969affb%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: b9a8ef19757862699609280067461b833ae1060a
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=451abbc4-25fe-4143-b1ef-88ddd5d77660%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/jumpstart/payload_utils.py (hunk 3 above)
Comment (evakravi) [reply src/sagemaker/jumpstart/payload_utils.py:118]: sorry, will do

File: src/sagemaker/jumpstart/utils.py (hunk 5 above)
Comment (evakravi) [reply src/sagemaker/jumpstart/utils.py:112]: for now, let's keep as is. i agree, there is code duplication that should be removed.

btw, this change is for ensuring the wildcard logging happens a single time. basically, what is happening is that when the model is initialized, the cache got reset, causing the wildcard warning to get emitted twice.

File: tests/unit/sagemaker/jumpstart/model/test_model.py (hunk 6 above)
Comment (evakravi) [reply tests/unit/sagemaker/jumpstart/model/test_model.py:176]: yeah let me see if i can simplify this, it was lifted from another test

Comment (evakravi) [reply tests/unit/sagemaker/jumpstart/model/test_model.py:176]: this is a different mocked session object we need to include. It's messy i know lol

File: src/sagemaker/jumpstart/exceptions.py [hunk 7]
```diff
@@ -154,8 +155,8 @@ def __init__(
                 raise RuntimeError("Must specify `model_id` and `version` arguments.")
             self.message = (
                 f"Version '{version}' of JumpStart model '{model_id}' is deprecated. "
-                "Please try targeting a higher version of the model or using a "
-                "different model."
+                "Please try targeting a higher version of the model, upgrading "
+                "SageMaker Python SDK version or using a different model."
```
Comment (JGuinegagne) [inline src/sagemaker/jumpstart/exceptions.py:159]: could you check with @judyheflin please? I think they generally don't like using `Please`.

File: src/sagemaker/jumpstart/types.py [hunk 8]
```diff
@@ -339,7 +339,6 @@ class JumpStartSerializablePayload(JumpStartDataHolderType):
         "content_type",
         "accept",
         "body",
-        "generated_text_response_key",
```
Comment (JGuinegagne) [inline src/sagemaker/jumpstart/types.py:342]: question: is this backward compatible?

File: src/sagemaker/model.py (hunk 1 above)
Comment (JGuinegagne) [reply src/sagemaker/model.py:651]: question: what happens if `accept_eula` is `False` ? Does it use the attribute?

Could you add a unit test for this case please?

File: src/sagemaker/model.py [hunk 9]
```diff
@@ -1434,7 +1454,12 @@ def deploy(
                 "serverless_inference_config needs to be a ServerlessInferenceConfig object"
             )
 
-        if instance_type and instance_type.startswith("ml.inf") and not self._is_compiled_model:
+        if (
+            getattr(self, "model_id", None) in {"", None}
+            and instance_type
+            and instance_type.startswith("ml.inf")
```
[the remark this answers is not in the document]
Comment (JGuinegagne) [reply src/sagemaker/model.py:1460]: can you create a separate utility for this + unit test it please?

File: tests/unit/sagemaker/model/test_model.py [hunk 10]
```diff
@@ -315,7 +315,25 @@ def test_create_sagemaker_model_accelerator_type(prepare_container_def, sagemake
...
+    model = Model(MODEL_IMAGE, MODEL_DATA, name=MODEL_NAME, sagemaker_session=sagemaker_session)
+
+    accelerator_type = "ml.eia.medium"
+    model.create(INSTANCE_TYPE, accelerator_type=accelerator_type, accept_eula=True)
+
+    prepare_container_def.assert_called_with(
+        INSTANCE_TYPE,
+        accelerator_type=accelerator_type,
+        serverless_inference_config=None,
+        accept_eula=True,
```
Comment (JGuinegagne) [inline tests/unit/sagemaker/model/test_model.py:336]: add a unit test for `False` case please

**Review by JGuinegagne — CHANGES_REQUESTED** [review]
Please review with tech writer and create additional utilities.

Also, you are removing an internal function, could you confirm no other team is using it?

File: src/sagemaker/jumpstart/types.py (hunk 8 above)
Comment (evakravi) [reply src/sagemaker/jumpstart/types.py:342]: yes.  we now have the `output_keys` field.

File: tests/unit/sagemaker/model/test_model.py (hunk 10 above)
Comment (evakravi) [reply tests/unit/sagemaker/model/test_model.py:336]: `question: what happens if accept_eula is False ? Does it use the attribute?`-- Good catch, I will fix this and add the test

`can you create a separate utility for this + unit test it please?` separate utility for what exactly? there already is a unit test that the log does not get emitted for JS models

File: src/sagemaker/model.py (hunk 1 above)
Comment (chrstfu) [reply src/sagemaker/model.py:651]: Yeah, totally agree @evakravi. Doesn't have to be this PR but can we put a backlog item to refactor `deploy()` to match and remove `getattr` & `self.accept_eula`?

File: src/sagemaker/jumpstart/utils.py (hunk 5 above)
Comment (chrstfu) [reply src/sagemaker/jumpstart/utils.py:112]: Ah I see, that makes a lot of sense. Nice catch!

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 6673e99cdd1345a1277d119676f7f9de144aeb74
* Result: FAILED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=4176abf2-d1a2-4861-8c15-f5b2d3660def%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 6673e99cdd1345a1277d119676f7f9de144aeb74
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=08a5350f-557c-4e13-9de3-38f8d563fb2b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: 6673e99cdd1345a1277d119676f7f9de144aeb74
* Result: FAILED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=7fc8a415-db5b-43fb-97a1-47b814df075e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 6673e99cdd1345a1277d119676f7f9de144aeb74
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=e4b3891d-4a7f-4f4d-bee2-244e9422ac21%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 86822d211b3194be5c25d7f49c807d6b79a68ebe
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=7d187a7f-d301-4b3e-9e83-8a04b276a005%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 86822d211b3194be5c25d7f49c807d6b79a68ebe
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=fd9c0350-7d56-4979-9b61-7b1a1716de24%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: ec0c76d2fecd3605a1ea510d47726c05afee48bb
* Result: SUCCEEDED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=f63bc009-a032-485e-a297-053645cf1023%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/jumpstart/exceptions.py (hunk 7 above)
Comment (judyheflin) [reply src/sagemaker/jumpstart/exceptions.py:164]: Recommendation: 
"Version '{version}' of JumpStart model '{model_id}' is deprecated. We recommend that you specify a more recent model version or choose a different model. To access the latest models and model versions, be sure to upgrade to the latest version of the SageMaker Python SDK."

**Review by chrstfu — APPROVED** [review]

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 3528e486a4f4573565032e0265fcc5c48db0b65e
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=5fd5d521-066d-44b3-83ba-db827a07cea0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 3528e486a4f4573565032e0265fcc5c48db0b65e
* Result: SUCCEEDED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=f370fc83-0553-4a65-927f-e4af7a362ac3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 3528e486a4f4573565032e0265fcc5c48db0b65e
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ed7ed44e-2df6-4d0f-82bb-4c580360231c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 0bcf25e3cab5790e6e2e20d03234c2a9b49649ab
* Result: FAILED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=585d09f4-748b-4f1c-bbda-fa216f865032%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: 0bcf25e3cab5790e6e2e20d03234c2a9b49649ab
* Result: SUCCEEDED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=4292cd3f-bb71-427d-905d-79e1c5e40709%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: 0bcf25e3cab5790e6e2e20d03234c2a9b49649ab
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=a8f0b392-0880-4946-a425-658d98989e8e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: 0bcf25e3cab5790e6e2e20d03234c2a9b49649ab
* Result: FAILED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=21e7a0c2-636d-4f2f-95ce-be7a532fa1e4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 0bcf25e3cab5790e6e2e20d03234c2a9b49649ab
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=de4dc8ee-dd51-4756-811f-0f8b511b9c9c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/sagemaker/jumpstart/types.py [hunk 11]
```diff
@@ -730,6 +730,7 @@ class JumpStartModelSpecs(JumpStartDataHolderType):
         "training_dependencies",
         "training_vulnerabilities",
         "deprecated",
+        "info_message",
```
Comment (JGuinegagne) [inline src/sagemaker/jumpstart/types.py:733]: Any way to use a more specific attribute name which makes it clear immediately to the reader what purpose it serves and when it gets displayed?

Comment (evakravi) [reply src/sagemaker/jumpstart/types.py:733]: sure, any suggestions?

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: 20b06f2b299fed0cbdcb074601385b3f60f26355
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=a121e481-a406-435c-8529-ea8a685b25f4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: 20b06f2b299fed0cbdcb074601385b3f60f26355
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=566f17eb-210d-4bf3-9c8b-0e369763912a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: 20b06f2b299fed0cbdcb074601385b3f60f26355
* Result: SUCCEEDED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=8a8f6f03-8b1a-41e8-ae8b-649411d4f2c8%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-local-mode-tests
* Commit ID: bca595562eaa514b3c298ca39aebd0f47dc45812
* Result: SUCCEEDED
* [Build Logs](https://8huhkqo601.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=67a23fd9-cea5-4d1a-a6c5-401635abf4a9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-pr
* Commit ID: bca595562eaa514b3c298ca39aebd0f47dc45812
* Result: SUCCEEDED
* [Build Logs](https://abkt2mlh3k.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=3d26969c-af55-4e56-b365-25565227da91%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-notebook-tests
* Commit ID: bca595562eaa514b3c298ca39aebd0f47dc45812
* Result: SUCCEEDED
* [Build Logs](https://bkqkixfb59.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=fdb7d9f1-3a50-413f-8b3b-07f1253952d6%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-slow-tests
* Commit ID: bca595562eaa514b3c298ca39aebd0f47dc45812
* Result: SUCCEEDED
* [Build Logs](https://omfrydptb9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=d8b66bb9-6541-4f4f-ac05-1a20296e0573%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by mufaddal-rohawala** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: sagemaker-python-sdk-unit-tests
* Commit ID: bca595562eaa514b3c298ca39aebd0f47dc45812
* Result: SUCCEEDED
* [Build Logs](https://fegcnxu6da.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ebcebd1f-0109-468c-8c4d-f27b2b1e8d3d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by JGuinegagne — APPROVED** [review]

**Review by knikure — APPROVED** [review]

**Review by knikure — APPROVED** [review]

**Pull request merged**